iOS/CodiumUnWrapper/CodiumUnWrapper.swift

```diff
@@ -0,0 +1,155 @@
+// Codium UnWrapper
+
+import Foundation
+import UIKit
+
+extension Optional {
+
+    func unWrapOr<T>(_ value : T) -> T {
+        if let val = self as? T {
```

Comment:
I like guard better you don't need else
```swift
guard let val = self as ? T { return value }
return val
```

Reply:
even the guard let needs else as well but if it's more readable I will use that . usually guard let will use when there is nothing to return (just return with nothing else), but anyways both of them will working similarly